src/core_plugins/kibana/server/tutorials/apm/index.js

```diff
@@ -0,0 +1,24 @@
+import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
+import { ON_PREM_INSTRUCTIONS } from './on_prem';
+import { ELASTIC_CLOUD_INSTRUCTIONS } from './elastic_cloud';
+import { ON_PREM_ELASTIC_CLOUD_INSTRUCTIONS } from './on_prem_elastic_cloud';
+
+export function apmSpecProvider() {
+  return {
+    id: 'apm',
+    name: 'APM',
+    category: TUTORIAL_CATEGORY.OTHER,
+    shortDescription: 'APM (Application Performance Monitoring) automatically collects in-depth' +
+      ' performance metrics and errors from inside your applications.',
+    longDescription: 'APM consists of three components - the Agents, the Server, and the UI:\n' +
+      '* The Agents are libraries in your application that run inside of your application process.\n' +
+      '* The Server processes data from agents and stores the application data in Elasticsearch.\n' +
+      '* The UI is this dedicated Kibana APM plugin and customizable dashboards.\n\n' +
```

Comment:
@nreese This sentence should probably be changed from 

>The UI is this dedicated Kibana APM plugin and customizable dashboards.

to

>The UI is the preconfigured dashboards.

Unless we want to mention / upsell the X-Pack stuff here. Not sure what others are doing. (cc @alexfrancoeur)

Reply:
Btw, is there a "Launch dashboard" button  in the end of the instructions? How is that configured?

Reply:
The launch dashboard button is displayed when [artifacts](https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/common/tutorials/tutorial_schema.js#L15) is provided by the manifest file with `dashboards` key properly filed out.